From eac0a6d09227a0fdaf503364c89aa0ee52560759 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 22 Feb 2024 10:57:10 +0100
Subject: [PATCH] arm64: defconfig: add stm32 timers support in addons fragment

Enable timer drivers that can be used on STM32MP25, as module.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I22b864a424a63fc15a4bcda886cdcd64d1eb76af
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361424
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,10 +1,14 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
 CONFIG_DYNAMIC_DEBUG=y
+CONFIG_IIO_STM32_TIMER_TRIGGER=m
+CONFIG_MFD_STM32_TIMERS=m
 CONFIG_OF_OVERLAY=y
+CONFIG_PWM_STM32=m
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REMOTEPROC=y
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
 CONFIG_STM32_RPROC=y
+CONFIG_STM32_TIMER_CNT=m
 CONFIG_TEE_REMOTEPROC=y
 CONFIG_USB_HIDDEV=y
